src/XR/native/nativeXRFrame.ts

```diff
@@ -0,0 +1,98 @@
+import { NativeXRPlugin } from "./nativeXRPlugin";
+
+/** @hidden */
+interface INativeXRFrame extends XRFrame {
+    // Native supported functions
+    createAnchor(pose: XRRigidTransform, space: XRSpace): Promise<XRAnchor>;
+    fillPoses(spaces: XRSpace[], baseSpace: XRSpace, transforms: Float32Array): boolean;
+    getJointPose(joint: XRJointSpace, baseSpace: XRSpace): XRJointPose;
+    fillJointRadii(jointSpaces: XRJointSpace[], radii: Float32Array): boolean;

```

Comment:
These are already part of the `XRFrame` interface right? I'm confused why they seem to be repeated here.

Reply:
This implies that they're implemented as opposed to optionally implemented (in webxr.d.ts they're listed with a `?`). I was just trying to avoid using `!` when calling the functions through `this._native`, but if it seems redundant I can take it out.

Reply:
Ah ok, got it.